test/e2e/framework/pod/create.go

```diff
@@ -33,6 +33,21 @@ var (
 	BusyBoxImage = imageutils.GetE2EImage(imageutils.BusyBox)
 )
 
+// PodInfo is a struct containing all arguments for creating a pod.
+// SELinux testing requires to pass HostIPC and HostPID as booleansi arguments.
+type PodInfo struct {
+	ns                  string
+	pvclaims            []*v1.PersistentVolumeClaim
+	inlineVolumeSources []*v1.VolumeSource
+	isPrivileged        bool
+	command             string
+	hostIPC             bool
+	hostPID             bool
+	seLinuxLabel        *v1.SELinuxOptions
```

Comment:
We can consider this in a followup, but I think we can potentially make these not arguments and just set them to some test value internally. @gnufied @jsafrane do you see a reason why a test would need to configure different fsgroup/selinux labels?